server/app/api.py

```diff
@@ -1227,72 +1194,6 @@ def score(self, obj, user, data):
         subm.put()
         return score
 
-    def get_assignment(self, name):
```

Comment:
Actually, it was just moved a couple dozen lines upward haha. I didn't remove it.